awsx/schema-types.ts

```diff
@@ -793,15 +793,15 @@ export interface TaskDefinitionContainerDefinitionOutputs {
     readonly firelensConfiguration?: pulumi.Output<TaskDefinitionFirelensConfigurationOutputs>;
     readonly healthCheck?: pulumi.Output<TaskDefinitionHealthCheckOutputs>;
     readonly hostname?: pulumi.Output<string>;
-    readonly image?: pulumi.Output<string>;
+    readonly image: pulumi.Output<string>;
     readonly interactive?: pulumi.Output<boolean>;
     readonly links?: pulumi.Output<string[]>;
     readonly linuxParameters?: pulumi.Output<TaskDefinitionLinuxParametersOutputs>;
     readonly logConfiguration?: pulumi.Output<TaskDefinitionLogConfigurationOutputs>;
     readonly memory?: pulumi.Output<number>;
     readonly memoryReservation?: pulumi.Output<number>;
     readonly mountPoints?: pulumi.Output<TaskDefinitionMountPointOutputs[]>;
-    readonly name?: pulumi.Output<string>;
```

Comment:
Can you explain why this changed?

Reply:
The `name` and `image` fields in upstream AWS Native now [marks these as required fields](https://www.pulumi.com/registry/packages/aws-native/api-docs/ecs/taskdefinition/#taskdefinitioncontainerdefinition).